src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
+
+The most frequently used directives focused on search engines and their paid marketing counterparts. SEO tools Ahref and Majestic were in the top five `Disallow` statements for both devices.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+          <th></th>
+          <th colspan="2">% of <code>robots.txt</code></th>
+      </tr>
+      <tr>
+        <th><code>User-agent</code></th>
+        <th>Mobile</th>
+        <th>Desktop</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>*</code></td>
+        <td class="numeric">74.40%</td>
+        <td class="numeric">73.16%</td>
+      </tr>
+      <tr>
+        <td>adsbot-google</td>
+        <td class="numeric">5.63%</td>
+        <td class="numeric">5.68%</td>
+      </tr>
+      <tr>
+        <td>mediapartners-google</td>
+        <td class="numeric">5.55%</td>
+        <td class="numeric">3.83%</td>
+      </tr>
+      <tr>
+        <td>mj12bot</td>
+        <td class="numeric">5.49%</td>
+        <td class="numeric">5.30%</td>
+      </tr>
+      <tr>
+        <td>ahrefsbot</td>
+        <td class="numeric">4.80%</td>
+        <td class="numeric">4.66%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="<code>robots.txt User-agent</code> directives.",
+      sheets_gid="243594173",
+      sql_file="pages_robots_txt_by_device_and_useragent.sql"
+    ) }}
+  </figcaption>
 </figure>
 
+When analyzing the usage of the `Disallow` statement in `robots.txt` by using Lighthouse-powered data of over 6 million sites, it was found that 97.84% of them were completely crawlable, with only 1.05% using a `Disallow` statement.
 
+An analysis of the `robots.txt` `Disallow` statement usage along the [meta robots](https://developers.google.com/search/reference/robots_meta_tag) _indexability_ directives was also done, finding 1.02% of the sites including a `Disallow` statement along indexable pages featuring a meta robots `index` directive, with only 0.03% of sites using the `Disallow` statement in `robots.txt` along _noindexed_ pages via the meta robots `noindex` directive.
 
-When analyzing the usage of the disallow statement in robots.txt by using Lighthouse-powered data of over 6 million sites, it was found that 97.84% of them were completely crawlable, with only 1.05% using a disallow statement.
+This is notable as [Google documentation](https://developers.google.com/search/docs/advanced/robots/intro) states that site owners should not use `robots.txt` as a means to hide web pages from Google Search, as internal linking with descriptive text could result in the page being indexed without a crawler visiting the page. Instead, site owners should use other methods, like a `noindex` directive via meta robots.
+{# TODO(authors): Tie this notable fact back to the data: is it notable because the disallow numbers are so low? What does that say about site owners following Google's guidance? #}
```

Comment:
@aleyda

Reply:
@rviscomi it's notable because of Google's guidance about not using robots.txt for that purpose and it is already tied to the "source" by linking to the Google documentation where these guidelines are given using the "Google Documentation" text as anchor text. 

The URL is: https://developers.google.com/search/docs/advanced/robots/intro where you can see the guidance: 

"You should not use robots.txt as a means to hide your web pages from Google Search results."

--- If this is not clear, could you please let me know how to better rephrase it and link to make it clearer?... I thought it was :(

Reply:
Thanks @rviscomi ! Understood :D I added the context on why adding disallow along more "indexable" pages rather than "noindexed" is what is notable.